tests/suites/test_suite_ssl.data

```diff
@@ -4154,6 +4154,10 @@ Record crypt, AES-128-GCM, 1.2
 depends_on:MBEDTLS_AES_C:MBEDTLS_SSL_PROTO_TLS1_2:MBEDTLS_GCM_C
 ssl_crypt_record:MBEDTLS_CIPHER_AES_128_GCM:MBEDTLS_MD_MD5:0:0:MBEDTLS_SSL_MINOR_VERSION_3:0:0
 
+Record crypt, AES-128-GCM, 1.3
+depends_on:MBEDTLS_AES_C:MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL:MBEDTLS_GCM_C
+ssl_crypt_record:MBEDTLS_CIPHER_AES_128_GCM:MBEDTLS_MD_MD5:0:0:MBEDTLS_SSL_MINOR_VERSION_4:0:0
```

Comment:
Suggestion - `MBEDTLS_MD_MD5`? GSM is AEAD and checking the source I don't think is used, so if we must pass a nonsense parameter, wouldn't `MBEDTLS_MD_NONE` be better? Using MD5 looks wrong, and made me check the source afterall.

Reply:
You're right. On the other hand, this is a fault in the entire SSL record checking test suite, and I'd prefer to fix it in a separate PR than to blow up this one (at least judging from cheer line count). OK?

Reply:
Agreed. PR's that are too big are so hard to get merged.